### ISSUE TITLE
On OSX agent console, info command error

### ISSUE THREAD

**Issue by hunter-nl** [Ylianst/MeshCentral#318]
On OSX agent console, info command results into error:
```
> info
Command returned an exception error: darwin not supported
```

**Comment by Ylianst** [comment]
That is one for Bryan, I suspect I know what is going on, but I need to chat with Bryan to figure out the proper fix for this.

**Comment by krayon007** [comment]
Ok, this is fixed in GIT. Just a minor change to the JavaScript. It was trying to check X11 status if the platform wasn't windows. I fixed it so it only does that if it's linux or freebsd.

**Comment by Ylianst** [comment]
Published MeshCentral v0.3.6-v with Bryan's fix. Please confirm and close if it works.

**Comment by hunter-nl** [comment]
Yes, fixed. THANKS!
